_docs/operations/updates.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: guide
+title: StorageOS Docs - Update StorageOS version
+anchor: operations
+module: operations/updates
+---
+
+# Update StorageOS version
+
+> Before upgrading, check the 
+> [release notes]({%link _docs/reference/release_notes.md %}) to confirm
+> whether there is a safe upgrade path between versions.
+
+
+StorageOS version upgrades must be planned and executed taking in consideration
+that data will remain inaccessible during the process. It is recommended to
+schedule a maintenance window.
+
+There are two strategies to update StorageOS.
+
+1. Full stop of the cluster
+1. Rolling update
+
+## Option 1. Full stop of the cluster
+
+This option consists on downscaling all applications using StorageOS volumes
+to 0, stop all StorageOS pods, start StorageOS with new version and rescale
+applications to previous size.
+
+This option will not require moving data across nodes, therefore it is
+recommended for clusters with large data sets. However, __it implies downtime__ of
+the stateful applications.
+
+Follow next steps to perform this upgrade
```

Comment:
This sentence doesn't parse well. Can remove entirely, or use something like 'Execute the following instructions:'

Reply:
done